### ISSUE TITLE
possible bug? 

### ISSUE THREAD

**Issue by skygragon** [twitter/twemproxy#673]
why add continuum_addition again? seems nserver_continuum already take it in count

![image](https://user-images.githubusercontent.com/1295096/185744128-f3499162-d68d-43ab-a4ad-b3e5f519ccaa.png)

**Comment by TysonAndre** [comment]
Yeah, it looks like continuum_addition (`*_CONTINUUM_ADDITION`) was already added, so the logged number of slots allocated (e.g. with erealloc) is wrong
